forgerock-auth/src/main/java/org/forgerock/android/auth/FRAuth.java

```diff
@@ -47,6 +47,8 @@ public static synchronized void start(Context context, @Nullable FROptions optio
         if(!started || !FROptions.equals(cachedOptions, options)) {
             started = true;
             FROptions currentOptions = ConfigHelper.load(context, options);
+            //Validate (AM URL, Realm, CookieName) is not Empty. If its empty will throw IllegalArgumentException.
+            currentOptions.validateConfig();
```

Comment:
Are we throwing the new exception here?

Reply:
yes , we are throwing a new exception, its intentional when the user supplies empty url, realm, cookiename